src/Tizen.NUI/src/internal/WebView/WebHttpRequestInterceptor.cs

```diff
@@ -46,17 +51,43 @@ public string Url
         {
             get
             {
-                return Interop.WebHttpRequestInterceptor.GetUrl(SwigCPtr);
+                return Interop.WebHttpRequestInterceptor.GetUrl(interceptorHandle);
+            }
+        }
+
+        /// <summary>
+        /// Gets method of intercepted http request, for example, GET, POST, etc.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Method
+        {
+            get
+            {
+                return Interop.WebHttpRequestInterceptor.GetMethod(interceptorHandle);
+            }
+        }
+
+        /// <summary>
+        /// Gets headers of intercepted http request, for example, "schema: https".
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public PropertyMap Headers
+        {
+            get
+            {
+                IntPtr mapPtr = Interop.WebHttpRequestInterceptor.GetHeaders(interceptorHandle);
+                return new PropertyMap(mapPtr, true);
             }
         }
 
         /// <summary>
         /// Ignores this request.
+        /// This function can be called only INSIDE WebContext.HttpRequestIntercepted.
```

Comment:
https://docs.tizen.org/application/native/api/mobile/5.0/group__WEBVIEW.html#gae484b204d142e0d117326e5683194f39
> Handling a request is done after using either ewk_intercept_request_ignore, ewk_intercept_request_response_set, ewk_intercept_request_response_body_set, and in some cases after ewk_intercept_request_response_write_chunk.
Further EWK API calls on this Ewk_Intercept_Request instance result in undefined behavior.

The ewk documentation says after calling one of 4 APIs, using the instance would make undefined behavior.
I think we need to specify this restriction.

Reply:
Done.